use-timescale/integrations/query-admin/find-connection-details.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Connecting to Timescale
+excerpt: How to connect to a Timescale instance
+products: [cloud, mst, self_hosted]
+keywords: [connect, Managed Service for TimescaleDB, Timescale]
+---
+
+# Find your connection details 
+
+To connect to your $SERVICE_SHORT or self-hosted database, you need at least the following:
+
+- Hostname
+- Port
+- Username
+- Password
+- Database name
+
+Find the connection details based on your installation type.
+
+<Tabs label="Connection details">
+
+<Tab title="Timescale Cloud">
+
+The connection details for your $SERVICE_LONG are available in $CONSOLE. Get them in one of the following ways:
```

Comment:
To retrieve the connection details for your $SERVICE_LONG:

* **In `timescale-<service name>-credentials.txt`**:
   All connection details are supplied in the configuration file you download when you create a new $SERVICE_SHORT. 
* **In $CONSOLE**:
   Open the [`Services`][console-services] page and select your $SERVICE_SHORT. The connection details, except the password, are available in `Connect to your service` widget. If necessary, click `Forgot your password?` to get a new one.
     Can you add a screenshot please.